calendarScript.js

```diff
@@ -0,0 +1,74 @@
+const date = new Date();
+
+const renderCalendar = () => {
+
+    date.setDate(1);
+
+    const monthDays = document.querySelector(".days");
+
+    const lastDay = new Date(date.getFullYear(),
+    date.getMonth() + 1, 0).getDate();  // 0 = last day of pre month, 1 = first day of current month //
+
+    const prevLastDay = new Date(date.getFullYear(),
+    date.getMonth(), 0).getDate();
+
+    const firstDayIndex = date.getDay();
+    const lastDayIndex = new Date(date.getFullYear(),
+    date.getMonth() + 1, 0).getDay();
+
+    const nextDays = 7 - lastDayIndex - 1;
+
+
+    const months = [
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December"
+    ];
+
+    document.querySelector(".date h1").innerHTML = months[date.getMonth()];
+
+    document.querySelector(".date p").innerHTML = new Date().toDateString();
+
+    let days ="";
+
+    for(let x = firstDayIndex; x >0; x--) {
```

Comment:
```suggestion
    for (let x = firstDayIndex; x > 0; x--) {
```

for, if 문과 같은 조건문과 반복문의 경우 여는 괄호 앞에 공백을 추가하는게 일반적입니다.

- https://github.com/airbnb/javascript#whitespace--around-keywords